eclipse-theia/theia
# [built-in extensions] Updated builtin extensions

**What it does**

Fixes: https://github.com/eclipse-theia/theia/issues/7416
Fixes:  https://github.com/eclipse-theia/theia/issues/7433

The following commit updates the example-applications to use builtin extensions from the new `open-vsx` instead of GitHub releases, and also updates the URLs so they are less fragile.

The following builtins were updated to their `open-vsx` counterparts:
1. node-debug
2. node-debug2
3. references-view

#### How to test
1. run command yarn on root.
2. run command rm -rf plugins/ (this will remove all the plugins)
3. run command yarn run download:plugins (this will download all the plugins)
4. check to see if the mentioned plugins are downloaded or not

#### Review checklist


#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

Signed-off-by: Anas Shahid <muhammad.shahid@ericsson.com>
